src/java/main/org/apache/zookeeper/cli/DeleteCommand.java

```diff
@@ -56,14 +56,7 @@ public CliCommand parse(String[] cmdArgs) throws CliParseException {
     }
 
     private void retainCompatibility(String[] cmdArgs) throws CliParseException {
-        // delete path [version]
         if (args.length > 2) {
-            // rewrite to option
-            String [] newCmd = new String[4];
```

Comment:
These are dead code. newCmd was declared but not used so it's better to remove them.